Gemfile

```diff
@@ -28,6 +28,9 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
 
+
+gem 'phonelib'
```

Comment:
Interesting, I've use [`phony`](https://github.com/joost/phony_rails) before for dealing with phone numbers, but this one also looks good!